stdlib/public/core/OptionSet.swift

```diff
@@ -278,7 +278,7 @@ extension OptionSet where Element == Self {
   ///     print(expressOption == .express)
   ///     // Prints "false"
   ///     print(expressOption == .secondDay)
-  ///     // Prints "true"
```

Comment:
This is indeed how OptionSet.remove works in 5.2, but that's actually a bug -- see https://github.com/apple/swift/pull/29436#discussion_r370906914

Reply:
Removed this change/commit on the PR against 'master'.  Will close this one.